charts/model-deployment/music-transformer/templates/servingruntime.yaml

```
@@ -18,7 +18,7 @@ spec:
     prometheus.io/port: '8888'
   containers:
     - args:
-        - '--model_name={{.Name}}'
+        - '--model_name={{ .Values.name }}'
```

hold on sorry, this one can stay as `{{.Name}}` , my mistake 😅

```suggestion
        - '--model_name={{.Name}}'
```